pkg/config/setup/config.go

```diff
@@ -2035,7 +2035,7 @@ func bindEnvAndSetLogsConfigKeys(config pkgconfigmodel.Config, prefix string) {
 	config.BindEnvAndSetDefault(prefix+"sender_backoff_base", DefaultLogsSenderBackoffBase)
 	config.BindEnvAndSetDefault(prefix+"sender_backoff_max", DefaultLogsSenderBackoffMax)
 	config.BindEnvAndSetDefault(prefix+"sender_recovery_interval", DefaultForwarderRecoveryInterval)
-	config.BindEnvAndSetDefault(prefix+"sender_recovery_reset", false)
+	config.BindEnvAndSetDefault(prefix+"sender_recovery_reset", true)
```

Comment:
Was changing this default intentional?

Reply:
yes I think it should be normal

Reply:
We should not change this without discussing with intake. The slow recovery is a feature, not a bug.

Reply:
Alright, I'll revert this change then